pykokkos/interface/__init__.py

```diff
@@ -49,7 +49,7 @@
     ScratchView, ScratchView1D, ScratchView2D,
     ScratchView3D, ScratchView4D, ScratchView5D,
     ScratchView6D, ScratchView7D, ScratchView8D,
-    from_cupy, from_numpy,
+    from_cupy, from_numpy, from_array,
```

Comment:
Do we want to even maintain explicit functions for diff arrays? @NaderAlAwar

Reply:
It might make sense to keep them internally but they should not be exposed to the user.

Reply:
I like that way more than exposing each of them

Reply:
I can hide them! from_array() (or whatever we are calling it) is just a wrapper to both of them anyways...